pkg/cli/validate.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"time"
+
+	"github.com/abcxyz/jvs/client-lib/go/client"
+	"github.com/spf13/cobra"
+)
+
+var (
+	flagToken           string
```

Comment:
I'm a little bit worried about supplying the token as a string, since then the JWT will appear in shell history. I wonder if we should support reading this from a pipe (e.g. stdin) or a raw string. For example:

```shell
jvsctl validate -token="ey2..."
```

would work as-is, but:

```shell
jvsctl validate -token=-
```

would read from stdin. That would allow callers to read the token from a file or envvar:

```shell
echo "${JVS_TOKEN}" | jvsctl validate -token=-
cat /tmp/jvs_token | jvsctl validate -token=-
```

Reply:
+1 I forgot to comment on this. In addition, I think the token should be an argument instead of a flag. `echo "${JVS_TOKEN}" | jvsctl validate -`

Reply:
I slightly prefer flags, because I prefer being explicit, but I'm largely indifferent.